cpp/build-support/cpplint.py

```diff
@@ -4008,9 +4008,9 @@ def CheckTrailingSemicolon(filename, clean_lines, linenum, error):
 
   # Block bodies should not be followed by a semicolon.  Due to C++11
```

Comment:
Can you open a PR into cpplint?

Reply:
Will do

Reply:
Done: https://github.com/google/styleguide/pull/562